TagsterWebAPI/src/Tagster.Infrastructure/Services/TagsService.cs

```diff
@@ -28,5 +29,10 @@ public async Task InstertDataAsync(Profile profile)
             await _tagsterDb.Profiles.AddAsync(profile);
             await _tagsterDb.SaveChangesAsync();
         }
+
+        public async Task<ProfileDto> GetHref(ProfileDto profileDto)
```

Comment:
maybe rename to GetByHref? Current name indicates that you will get href which is not true